src/main/resources/application.yml

```diff
@@ -24,8 +24,11 @@ spring:
     validationQuery: SELECT 1
 nakadi:
   topic:
+    min:
+      retentionMs: 86400000 # 1 days
     max:
       partitionNum: 8
+      retentionMs: 172800000 # 2 days
```

Comment:
I thought we would be using a larger `max` value. Teams are complaining that if they have a problem during the weekend, when they are back to work on monday events are already gone. We should have at least 3 days, so they have a change to fix it on Monday.